extensions/commands/art/cmd_build_info.py

```diff
@@ -389,7 +389,8 @@ def build_info_create(conan_api: ConanAPI, parser, subparser, *args):
     data = load_json(args.json)
 
     # remove the 'conanfile' node
-    data["graph"]["nodes"].pop("0")
+    if not data["graph"]["nodes"]["0"]["prev"]:
```

Comment:
what is the meaning of "prev" here? Like _previous_ node in the graph? _Previous_ dependency? Maybe it could be useful to update the comment in the upper line with a small clarification

Reply:
Yes, I think it would be good to add a comment there, `prev` it's the field that stores the package revision in the graph json output, no `prev` means that it has no binaries

Reply:
Added a comment there